### ISSUE TITLE
EscrowFinish fees may need updating on xrpl.org

### ISSUE THREAD

**Issue by sappenin** [XRPLF/xrpl-dev-portal#2915]
The description of on xrpl.org of [EscrowFinish](https://github.com/XRPLF/xrpl-dev-portal/blob/44412e1c62dc96e813fc3788e9d1abef8014dfc2/docs/references/protocol/transactions/types/escrowfinish.md#L4) fees/costs assumes a 10 drop ledger base fee. However, many validators are [voting for 200 drops](https://xrpscan.com/validators), so this value may change in the future. 

Here's a helpful link for how rippled computes the EscrowFinish fee (at least, for escrows that have Fulfillments): https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/Escrow.cpp#L368

**Comment by mDuo13** [comment]
Yeah, the [special transaction costs table](https://xrpl.org/docs/concepts/transactions/transaction-cost#special-transaction-costs) will also need to be updated along with probably lots of other places if a 200-drops base fee change goes through.

**Comment by DennisDawson** [comment]
Fixed in https://github.com/XRPLF/xrpl-dev-portal/pull/3131
